src/components/elements/favorite-button.tsx

```diff
@@ -17,9 +17,12 @@ const FavoriteButton = ({uuid, title, path, units, ...props}: Props) => {
       type="button"
       role="switch"
       aria-checked={isFavorite}
-      aria-labelledby={uuid}
       {...props}
     >
+      <span className="sr-only">
+        {isFavorite ? "Add" : "Remove"}
+        {` favorite ${title}`}
```

Comment:
Need to switch the logic.
```suggestion
        {isFavorite ? `Remove favorite "${title}"` : `Add favorite "${title}"`}
```